layouts/default.vue

```diff
@@ -16,7 +16,7 @@ export default {
 </script>
 
 <style>
-.page {
+.content {
```

Comment:
Adds a minimum height so that blank pages still take up the majority of the page instead of sandwiching all of the content to the top of the page in descending order